openmc/mgxs/groups.py

```diff
@@ -54,7 +54,7 @@ def __deepcopy__(self, memo):
     def __eq__(self, other):
         if not isinstance(other, EnergyGroups):
             return False
-        elif self.group_edges != other.group_edges:
+        elif (self.group_edges != other.group_edges).all():
```

Comment:
I..... dont know why I made that change and I cant think of a reason why that could possibly be needed.